Makefile

```diff
@@ -24,15 +28,21 @@ create_admin:
 	$(RUN_MANAGE) createsuperuser --username admin
 
 backend:
-	$(COMPOSE_CMD) run -d -p 8000:8000 --rm django
+	$(COMPOSE_CMD) run -p 8000:8000 --rm django
 
 client:
 	make backend
 	make frontend
 	make start
 
 frontend:
-	docker build -t django-trench-frontend ./examples/client/
+	docker build -d -t django-trench-frontend ./examples/client/
 
 start:
 	docker run -p 3000:3000 django-trench-frontend
+
+isort:
+	$(foreach dir,$(SOURCE),cd $(dir); $(ISORT))
+
+flake8:
+	$(foreach dir,$(SOURCE),cd $(dir); $(FLAKE8))
```

Comment:
Do we really need foreach for flake8 and isort? We should skip migrations for flake8